bundles/org.openhab.ui/web/src/pages/settings/rules/script/blockly-editor.vue

```diff
@@ -302,6 +309,7 @@
             </shadow>
           </value>
         </block>
+        <block type="oh_color_to_hsb" />
```

Comment:
Can you add a colour_picker shadow block attached to the input?

Reply:
done